calico-cloud/image-assurance/exclude-vulnerabilities-from-scan-results.mdx

```diff
@@ -0,0 +1,81 @@
+You may need to take corrective action to stabilize your workflow.
+- **Maximum CVSS score:** An image’s maximum CVSS score may be reduced to a lower score.
+An exception could eliminate the vulnerability with the highest CVSS score for an image.
+- **Scan assessment:** The assessment value (Pass, Warn, or Fail) could change because it is based on the maximum CVSS score.
+For example, Fail could change to Warn, and Warn could change to Pass.
+- **Vulnerability global alerts:** Alerts may no longer be triggered.
+Alerts are triggered based on scan results or maximum CVSS score values of images.
+- **Admission controller policies:** Pods could be created where they were previously blocked.
+Admission controller policies are based on vulnerability information (scan result or maximum CVSS score).
+
+:::
+
+## Exclude a vulnerability from your scan results
+
+You can exclude a vulnerability from your scan results page.
+
+1. From the Calico Cloud Manager UI, go to **Image Assurance > All Scan Results**.
+1. Select an image from the list.
+1. On the scan results panel that appears, expand a package, and then click **Actions > Add Exception**.
+1. On the **Add exception** dialog, select a scope, enter an explanation for why you're creating an exception, and then click **Save**.
+
+## Exclude multiple vulnerabilities at the same time
+
+To exclude large numbers of vulnerabilities, you can organize the required information as a CSV file and import it directly to Calico Cloud.
+
+1. Optional: You can start with a preformatted list of all vulnerabilities in your cluster, and then edit the list to include only those vulnerabilities you want to exclude.
+1. From the Calico Cloud Manager UI, go to **Image Assurance > All Scan Results**.
+1. On the **All Scan Results** page, click the **Export** button, and then click **Export data**.
+1. On the **Export** dialog that appears, select the **CSV** data type and the **Export a row for each image and CVE IDs** option for the table style.
+Click **Export** to download the CSV file.
+1. Edit the CSV file
+
+1. Prepare the CSV file with information about the vulnerabilities you want to exclude.
+The file must contain the following information:
+
+| Column header | Description | Example |
+   | -- | -- | -- |
+| `CVE` | The CVE identifier. Required by the `any`, `repo`, and `image` scopes. | `CVE-2024-1234` |
+| `Registry`| The URL path to the container registry. Required by the `repo` and `image` scopes. | `mycontainerregistry.io/my-org` |
+| `Repository`| The container image name. Required by the `repo` and `image` scopes.| `my-application` |
+| `Tags`| A JSON list of image tags. Required by the `image` scope. | `"[""v1.2.3"",""v2.3.4""]"` |
+| `Justification` | Your explanation for why you're excluding this vulnerability. Required by the `any`, `image`, and `image` scopes.| `This one is a false positive.` |
```

Comment:
Hey @ctauchen , I believe that there is a typo here. Should it be repo instead of image?